elifetools/utils.py

```diff
@@ -252,6 +273,14 @@ def remove_doi_paragraph(tags):
     return p_tags
 
 
+def lazy_remove_doi_paragraph(tags):
```

Comment:
I'm not sure this is used in `parseJATS.py`, the original `remove_doi_paragraph()` is still in use there?